src/app/containers/Blocks/index.test.jsx

```diff
@@ -54,4 +54,13 @@ describe('Blocks', () => {
     'should render correctly',
     <Blocks blocks={blocks} componentsToRender={componentsToRender} />,
   );
+
+  shouldShallowMatchSnapshot(
```

Comment:
This test can be removed based on this comment https://github.com/BBC-News/simorgh/pull/874/files#r229646193